packages/bpk-tokens/src/android/base/aliases.json

```diff
@@ -41,7 +41,9 @@
     "BORDER_SIZE_SM": 1,
     "BORDER_SIZE_LG": 2,
     "BORDER_SIZE_XL": 3,
+    "BORDER_RADIUS_XS": 4,
     "BORDER_RADIUS_SM": 4,
+    "BORDER_RADIUS_MD": 12,
```

Comment:
So this unblocks the card/panel change, but not the [button change](https://gojira.skyscanner.net/browse/BPK-4232). Is that correct?

Reply:
Yes that is correct

Reply:
Yes that is correct as we are not immediately working on the button change we don't want to make the breaking change until we are ready to work on that and adopt it - as if any other urgent fixes are needed before that work starts it would mean we are forced to adopt the breaking change